components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/java/org/wso2/carbon/identity/api/server/idp/v1/core/ServerIdpManagementService.java

```diff
@@ -2768,6 +2772,11 @@ private void processPatchRequest(List<Patch> patchRequest, IdentityProvider idpT
                                     Constants.ErrorMessage.ERROR_CODE_INVALID_CERTIFICATE_FORMAT, null);
                         }
                     }
+                    if (certificates.contains(value)) {
+                        throw handleException(Response.Status.CONFLICT,
+                                Constants.ErrorMessage.ERROR_CODE_ERROR_UPDATING_IDP,
```

Comment:
Can you check the formatting?

Reply:
Rechecked with codestyles. no issues found.